examples/pip_repository_annotations/pip_repository_annotations_test.py

```diff
@@ -64,6 +64,34 @@ def test_copy_executables(self):
         stdout = proc.stdout.decode("utf-8").strip()
         self.assertEqual(stdout, "Hello world from copied executable")
 
+    def test_data_exclude_glob(self):
+        current_wheel_version = "0.37.1"
+
+        r = runfiles.Create()
+        dist_info_dir = (
+            "pip_repository_annotations_example/external/{}/wheel-{}.dist-info".format(
+                self.wheel_pkg_dir(),
+                current_wheel_version,
+            )
+        )
+
+        # Note: `METADATA` is important as it's consumed by https://docs.python.org/3/library/importlib.metadata.html
+        # `METADATA` is expected to be there to show dist-info files are included in the runfiles.
+        metadata_path = r.Rlocation("{}/METADATA".format(dist_info_dir))
```

Comment:
This is a restored regression test from https://github.com/bazelbuild/rules_python/pull/626 but has new meaning as it now uses `METADATA` files to ensure this isn't removed from the `data` attributes of generated targets.